packages/client/src/hooks/useSlug.ts

```diff
@@ -5,7 +5,7 @@ import { getSlugFromPathname } from "../utils";
 export default (): string => {
   const { pathname } = useLocation();
   const slug = getSlugFromPathname(pathname);
-  if (!slug) {
+  if (!slug && pathname !== "/") {
```

Comment:
if pathname === "/" then there is no slug. Why add this check here?

Reply:
The DevHomePage has no slug and is not accessible because of this `throw Error()`. Or shall I remove this check for `!slug`?